### ISSUE TITLE
Evolve hyperparameters 

### ISSUE THREAD

**Issue by heyzx123** [ultralytics/yolov5#10418]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

hi,I would like to ask why when I use hyperparameter evolution, I get a validation set loss of nan, but I get P,R, Map?

![1670326830(1)](https://user-images.githubusercontent.com/92321786/205902752-41f3131c-6eb9-4a04-97ce-96ca779b78c3.jpg)

**Comment by heyzx123** [comment]
When it comes up nan, the program stops with an error, I just have to change the code and let it continue to run and get the best set of hyperparameters?

**Comment by glenn-jocher** [comment]
@heyzx123 oh got it. If you develop a fix in the code please consider submitting a PR to assist other users in the future. Thanks!
